DESCRIPTION

```diff
@@ -86,6 +86,7 @@ Imports:
     countrycode (>= 1.2.0),
     dplyr,
     httr,
+    jsonlite,
```

Comment:
As noted elsewhere, we already depend on `jsonlite` implicitly because `httr` depends on it, so I think it should be fine for us to add it as an explicit import here.